aspnetcore/security/authentication/azure-ad-b2c-webapi.md

```diff
@@ -0,0 +1,208 @@
+
+## Run the web API
+
+In Visual Studio, press *F5* to run the API. Visual Studio launches a browser pointed at the APIs root URL. Note the URL in the address bar, and leave the API running in the background.
+
+> [!NOTE] Since there is no controller defined for the root URL, the browser displays a 404 (page not found) error. This is expected behavior.
+
+## Use Postman to get a token and test the API
+
+[Postman](https://getpostman.com/postman) is a tool for testing APIs. For this tutorial, Postman will simulate a web app that accesses the web API on the user's behalf.
+
+### Register Postman as a web app
+
+Since Postman is going to simulate a web app that can obtain tokens from the Azure AD B2C tenant, it must be registered in the tenant as a web app.  Register Postman using [the steps in the documentation](/azure/active-directory-b2c/active-directory-b2c-app-registration#register-a-web-app) under the **Register a web app** section. Stop at the **Create a web app client secret** section. A client secret isn't required for this tutorial. 
+
+Use the following values:
+
+| Setting                       | Value                            | Notes                           |
+|-------------------------------|----------------------------------|---------------------------------|
+| **Name**                      | Postman                          |                                 |
+| **Include web app / web API** | Yes                              |                                 |
+| **Allow implicit flow**       | Yes                              |                                 |
+| **Reply URL**                 | `https://getpostman.com/postman` |                                 |
+| **App ID URI**                | *&lt;leave blank&gt;*            | Not required for this tutorial. |
+| **Include native client**     | No                               |                                 |
+
+The newly-registered web app needs permission to access the web API on the user's behalf.  
+
+1. Select **Postman** in the list of apps, and then select **API access** from the menu on the left.
+2. Select **+ Add**.
+3. In the **Select API** dropdown, select the name of the web API.
+4. In the **Select Scopes** dropdown, ensure all scopes are selected.
+5. Select **Ok**.
+
+Note the Postman app's Application ID, as it is required.
+
+### Create a Postman request
+
+Launch Postman. By default, Postman displays the **Create New** dialog upon launching. If the dialog is not displayed, select the **+ New** button in the upper left.
+
+From the **Create New** dialog:
+
+1. Select **Request**.
+    
+    ![Request button](./azure-ad-b2c-webapi/postman-create-new.png)
+
+2. Enter *Get Values* in the **Request name** box.
+3. Select **+ Create Collection** to create a new collection in which to store the request. Name the collection *ASP.NET Core tutorials* and then select the checkmark.
```

Comment:
~~in which to store~~
for storing

Reply:
My understanding is that commas kick in on 3+ clauses.  The instance you refer to uppage (formerly on 119, it's since moved) was a typo.

Reply:
Yes ... 3+ ... it was the two clause case that was tricky when the clauses are long. We might not (probably not, really) have been consistent across the whole repo. There was a discussion at some point in ancient times with Tom about this. IIRC the MS manual defaults to Chicago on it. IIRC Chicago says when the clauses are short to not sweat the comma in the two clause case. Where we may have a consistency issue is when the clauses are of the medium and long variety.